shared/src/business/useCases/processStreamRecordsInteractor.js

```diff
@@ -187,10 +212,26 @@ exports.processStreamRecordsInteractor = async ({
         records: recordsToProcess,
       });
 
-      for (const record of recordsToReprocess) {
-        try {
-          const newImage = record.dynamodb.NewImage;
+      const reIndexRecord = record => async e => {
+        await applicationContext
+          .getPersistenceGateway()
+          .createElasticsearchReindexRecord({
+            applicationContext,
+            recordPk: record.dynamodb.Keys.pk.S,
+            recordSk: record.dynamodb.Keys.sk.S,
+          });
 
+        applicationContext.logger.info(
```

Comment:
Same as above: can we use `logger.error`?

Reply:
Updated!